decidim-elections/app/commands/decidim/votings/check_census.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # A command to check if census data is given
+    class CheckCensus < Rectify::Command
+      def initialize(form)
+        @form = form
+      end
+
+      # Executes the command. Broadcast this events:
+      # - :ok when census is found
+      # - :invalid when the form is not valid
+      # - :not_found when census is not found
+      #
+      # Returns nothing.
+      def call
+        return broadcast(:invalid) unless form.valid?
+
+        check_census!
+      end
+
+      attr_reader :form, :session
+
+      def check_census!
+        if Decidim::Votings::Census::Datum.exists?(hashed_check_data: form.hashed_check_data)
```

Comment:
This should be scoped to this election or space maybe?

Reply:
Yes, I think so as same datum can be used fro diferent votings, but scoped to the `Dataset`
```suggestion
        if Decidim::Votings::Census::Datum.exists?(dataset: current_participatory_space.dataset, hashed_check_data: form.hashed_check_data)

```